sources/Renderer/OpenGL/Platform/GLSwapChainContext.h

```diff
@@ -52,7 +52,7 @@ class GLSwapChainContext
         // Creates a platform specific GLSwapChainContext instance.
         static std::unique_ptr<GLSwapChainContext> Create(GLContext& context, Surface& surface);
 
-        // Makes the specified swap-chain context link current. If null, no context is current.
+        // Primary function to make the specified swap-chain context link current.
```

Comment:
Is this comment change still valid? I guess it doesn't make a difference, but this doesn't mention the null pointer anymore. Same with the comment below. If this file doesn't need any changes, it also makes the PR a bit easier to navigate.